raco/algebra.py

```diff
@@ -861,6 +910,13 @@ def num_tuples(self):
             return 1
         return self.input.num_tuples()
 
+    def partitioning(self):
```

Comment:
I don't think this can work for the same reason I stated under `IdenticalSchemeBinaryOperator`.